mflowgen/soc/pad-frame-rtl/gen_rtl.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cp /sim/steveri/pad_frame/barebones/pad_frame.sv outputs/design.v
+cp /sim/steveri/pad_frame/barebones/io_file outputs/io_file
+
```

Comment:
You're still using my cached copy from way back? Okay. But at some point this will change, yes?

Reply:
Oops. This shouldn't be in the soc block flow. It doesn't need it, and the full chip flow uses the SoC repo, not your cached version. I'll remove this before merging.